workshops/views.py

```diff
@@ -496,6 +509,13 @@ def membership_create(request, org_domain):
     return render(request, 'workshops/generic_form.html', context)
 
 
+class MembershipCreate(OnlyForAdminsMixin, PermissionRequiredMixin,
+                 CreateViewContext):
+    permission_required = 'workshops.add_membership'
+    model = Membership
+    form_class = MembershipForm
```

Comment:
Perhaps it'd useful to have an autocompletion field `organization` in the `MembershipForm`.